package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nerdwallet/apollo-cache-policies",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
I'll make this a minor later, needed to bump it for packaging and testing locally for now